FileMetadataReading.py

```diff
@@ -0,0 +1,31 @@
+from datetime import datetime
+import os
+from FileDiscovery import FileDiscovery
+
+
+class FileMetadateRading:
+
+    def __init__(self, FiltratedListOfPhotos):
+        self.list_of_photos = FiltratedListOfPhotos
+
+    def read_modification_date_time(self):
+        time_md = [datetime.fromtimestamp(os.path.getmtime(i))
+                   for i in self.list_of_photos]
+        self.time_md_date_and_time = [
+            str(i.date()) + '-' + str(i.time()) for i in time_md]
+        return self.time_md_date_and_time
+
+
+class GenerateFullName:
```

Comment:
this class operates on list of files so name is wrong.

I recommend to rewrite this class so it works on one file instead.